core/frontend/src/internal/render/webgl/SceneCompositor.ts

```diff
@@ -320,16 +337,31 @@ class FrameBuffers implements WebGLDisposable {
 
   private initPotentialMSMRTFbos(textures: Textures, depth: DepthBuffer, depthMs: DepthBuffer | undefined): boolean {
     const boundColor = System.instance.frameBufferStack.currentColorBuffer;
-    assert(undefined !== boundColor && undefined !== textures.color && undefined !== textures.featureId && undefined !== textures.depthAndOrder && undefined !== textures.accumulation && undefined !== textures.revealage);
-    const colorAndPick = [boundColor, textures.featureId, textures.depthAndOrder];
+    assert(
+      undefined !== boundColor &&
+      undefined !== textures.color &&
+      undefined !== textures.featureId &&
+      undefined !== textures.depthAndOrder &&
+      undefined !== textures.contours &&
+      undefined !== textures.accumulation &&
+      undefined !== textures.revealage
+    );
+    const colorAndPick = [boundColor, textures.featureId, textures.depthAndOrder, textures.contours];
 
     if (undefined === depthMs) {
       this.opaqueAll = FrameBuffer.create(colorAndPick, depth);
       colorAndPick[0] = textures.color;
       this.opaqueAndCompositeAll = FrameBuffer.create(colorAndPick, depth);
     } else {
-      assert(undefined !== textures.colorMsBuff && undefined !== textures.featureIdMsBuff && undefined !== textures.featureIdMsBuffHidden && undefined !== textures.depthAndOrderMsBuff && undefined !== textures.depthAndOrderMsBuffHidden);
-      const colorAndPickMsBuffs = [textures.colorMsBuff, textures.featureIdMsBuff, textures.depthAndOrderMsBuff];
+      assert(
+        undefined !== textures.colorMsBuff &&
+        undefined !== textures.featureIdMsBuff &&
+        undefined !== textures.featureIdMsBuffHidden &&
+        undefined !== textures.contoursMsBuff &&
+        undefined !== textures.depthAndOrderMsBuff &&
+        undefined !== textures.depthAndOrderMsBuffHidden
+      );
+      const colorAndPickMsBuffs = [textures.colorMsBuff, textures.featureIdMsBuff, textures.depthAndOrderMsBuff, textures.contoursMsBuff];
       const colorAndPickFilters = [GL.MultiSampling.Filter.Linear, GL.MultiSampling.Filter.Nearest, GL.MultiSampling.Filter.Nearest];
```

Comment:
Looks like colorAndPickFilters should have a 4th entry in it for the new render target (nearest filter).

Reply:
D'oh. Thank you!